src/authorizations/users.rs

```diff
@@ -1,48 +1,27 @@
-fn authorized_users() -> Vec<String> {
-    dotenv::var("AUTHORIZED_USERS")
-        .unwrap()
-        .split(',')
-        .map(String::from)
-        .collect()
-}
+use crate::config::Config;
 
-pub fn is_authorized(id: &str) -> bool {
-   authorized_users().iter().any(|i| i == id)
+pub fn is_authorized(id: &str, config: &Config) -> bool {
+    config.authorized_users.iter().any(|i| i == id)
```

Comment:
Now that this is an `HashSet` you can simply use the contains method.

Reply:
ah! excellent point!